src/auth/screens/language-settings.js

```diff
@@ -7,4 +7,8 @@ export default [
     code: 'fr',
     name: 'Français',
   },
+  {
+    code: 'nl',
+    name: 'Dutch',
```

Comment:
@RolfKoenders there must be a native language name, if I'm not mistaken, then `Nederlands`, not `Dutch`?